src/frontend/styles/_typography.scss

```diff
@@ -119,22 +124,44 @@ h5,
 
 .subtitle,
 .content,
-.uppercase,
 p,
 h6,
-.text--md {
+.text--md,
+button,
+.button-md {
   font-size: var(--text-md);
 }
 
-.content-s,
-.button,
+.content-sm,
+.button-sm,
 .text--sm {
   font-size: var(--text-sm);
 }
 
-.subtitle-s,
+.subtitle-sm,
 .caption,
 .status,
 .text--xs {
   font-size: var(--text-xs);
 }
+
+.subtitle,
+.content-sm,
+h5,
+h4 {
+  font-weight: var(--medium);
+}
+
+.subtitle-sm,
```

Comment:
do we need these changes? we are not using these utility classes anywhere, and it's not clear when we'll use utility classes

Reply:
yeah we can remove all the utility classes if you want. Billie and I came up with these and are going to start trying to use them in new designs. I went ahead and added them so that other frontend devs could build with them. This way it prevents rework/refactoring later on

Reply:
what do you think? leave them in and start integrating them now or wait until later and do a more formal refactor?

Reply:
hmmm we don't have a definition on how to write the css so we currently have a mess of naming conventions and methodologies; we have camelCase, kebab-case, BEM, utility classes, in some places I even saw inline styles

unless it's clear what should be used in new code people might miss these new utility classes since the code doesn't instruct people to use them

I don't have a strong opinion on what we should aim for in heroic's css conventions (I personally don't like utility classes, but it's just a personal opinion) but I would say to keep these utility classes only if it's clear that we want to go in that direction of writing styles

Reply:
Gotcha. Yeah I think it would be good to try and standardize and restrict them to some predefined styles defined in a design system here and in the figma file. It'll make it easier to change later to have it defined in one place and help prevent the mess that you're describing.

It'll be a pretty big refactor to take the current text css styles and redefine them here. It should probably start in a figma file too, so I haven't started working on that (and why these are unused currently)

Reply:
I think we should move this discussion into an issue @arielj since it's not really related to this PR

Reply:
sure!